com.zandronum.Zandronum.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Categories=Games;
+Comment=Leading the way in newschool multiplayer Doom online
+Categories=Game;Shooter;
```

Comment:
The categories are duplicated here.

Reply:
First one is also wrong I now see. Must not be plural.